lib/structures/UnknownInteraction.js

```diff
@@ -37,7 +37,11 @@ class UnknownInteraction extends Interaction {
         }
 
         if(data.message !== undefined) {
-            this.message = new Message(data.message, this._client);
+            if(data.message.flags === InteractionFlags.EPHEMERAL) {
```

Comment:
this should use bit math instead of `===`, just in case other flags are present

Reply:
kinda invalidates my other one, but this one should be done either way

Reply:
Yea I agree, thought maybe I was stupid because it wasn't already done this way but

Reply:
Done in 32bfdb5